django_project/base/models/project.py

```diff
@@ -159,8 +159,8 @@ class Project(models.Model):
     )
 
     project_repository_url = models.URLField(
-        help_text=u'A repository URL for this project '
-                  u'e.g, a Github repository URL.',
+        help_text=u'A repository URL for this project. '
+                  u'For instance a path to the project\s Github repository.',
```

Comment:
As Tim mentioned, better to put bracket when you are doing multiline strings.
```
help_text=(
    u'Looooong string'),
```
Example: https://github.com/inasafe/inasafe/blob/develop/safe/definitions/layer_purposes.py#L86

Also `GitHub` not `github`: https://en.wikipedia.org/wiki/GitHub 
And also the apostrophe after project?